express/code/blocks/how-to-steps-carousel/how-to-steps-carousel.js

```diff
@@ -52,12 +52,16 @@ function activate(block, target) {
   // de-activate all
   block.querySelectorAll('.tip, .tip-number').forEach((item) => {
     item.classList.remove('active');
+    item.setAttribute('aria-selected', 'false');
   });
 
   // get index of the target
   const i = parseInt(target.getAttribute('data-tip-index'), 10);
   // activate corresponding number and tip
-  block.querySelectorAll(`.tip-${i}`).forEach((elem) => elem.classList.add('active'));
+  block.querySelectorAll(`.tip-${i}`).forEach((elem) => {
+    elem.classList.add('active');
+    elem.setAttribute('aria-selected', 'true');
```

Comment:
this is adding aria-selected=true to both .tip-number and .tip. It makes sense for .tip-number since it's selectable, but doesn't make sense for .tip as it's the controlled info. Maybe you actually want to do aria-expanded for them?